docs/components/Link/Web.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { Link } from "@jobber/components/Link";
+
+export default {
+  title: "Components/Text and Typography/Link/Web",
+  component: Link,
+  parameters: {
+    viewMode: "story",
+    previewTabs: { code: { hidden: false } },
+  },
+} as ComponentMeta<typeof Link>;
+
+const BasicTemplate: ComponentStory<typeof Link> = args => (
+  <Link {...args}>What is a Link anyway?</Link>
+);
+
+export const Horizontal = BasicTemplate.bind({});
```

Comment:
Horizontal seems like an odd name for this. There isn't a vertical variation and I'm not sure it's descriptive of this example as it just seems to be a basic link. Should we rename it?

Reply:
lol I copied it from `Divider` and forgot to change this const name, thanks for catching it

Reply:
ah ok sweet!